DBTableToHBaseTable/README.rst

```diff
@@ -0,0 +1,57 @@
+  2. ``connectionString`` - This is the JDBC conection string including the database name.
+  3. ``tableName`` - This is the table you wish to import from.
+  4. ``user`` - This is the username to use to connect to the specified database. It is required for databases that need authentication, optional for those that do not.
+  5. ``password`` - This is the password to use to connect to the specified database. If the database requires authentication, both username and password must be provided.
+  6. ``importQuery`` - This is the SELECT query to use to import data from the specified table. You can specify an arbitrary number of columns to import, or import all columns using *. You can also specify a number of WHERE clauses or ORDER BY clauses. However, LIMIT and OFFSET clauses should not be used in this query.
+  7. ``countQuery`` - This is the SELECT query to use to get the count of records to import from the specified table. Examples: SELECT COUNT(*) from <my_table> where <my_column> 1, SELECT COUNT(my_column) from my_table). NOTE: Please include the same WHERE clauses in this query as the ones used in the import query to reflect an accurate number of records to import.
+  8. ``jdbcPluginName`` - The name of the external JDBC plugin. This is the value of the ``name`` field in the external plugin's json configuration file. Defaults to 'jdbc'. Please find examples of external plugins at ``mysql-connector-java-5.1.35.json`` and ``postgresql-9.4.json``. Also refer to the CDAP documentation on external plugins for more details.
+  9. ``jdbcPluginType`` - The name of the external JDBC plugin. This is the value of the ``type`` field in the external plugin's json configuration file. Defaults to 'jdbc'. Please find examples of external plugins at ``mysql-connector-java-5.1.35.json`` and ``postgresql-9.4.json``. Also refer to the CDAP documentation on external plugins for more details.
+
+Configurations for the CDAP HBase Table Sink
+  1. ``name`` - This is the name of the HBase table to export to.
+  2. ``schema`` - This is the JSON representation of the HBase Table's schema
+  3. ``schema.row.field`` - This is the field in the ``StructuredRecord`` input to this Sink that would be used as the ``rowKey`` in the HBase table.
+
+::
+
+  cdap> create adapter dbIngest DBTableToHBaseTable/config.json
+  Successfully created adapter 'dbIngest'
+
+  cdap> start adapter dbIngest
+  Successfully started adapter 'dbIngest'
+
+You can verify that the data has been written to the HBase Table by viewing the contents of the HBase Table by executing the following CLI command:
+
+::
+
+  cdap> execute 'select * from dataset_<table_name>'
+
+You have now successfully created an Adapter that reads from a Database Table and writes to a CDAP HBase Table.
+
+You can stop and delete the Adapter using the CDAP CLI.
+
+::
+
+  cdap> stop adapter dbIngest
+  Successfully stopped adapter 'dbIngest'
+
+  cdap> delete adapter dbIngest
+  Successfully deleted adapter 'dbIngest'
```

Comment:
See other guides for closing text:

https://raw.githubusercontent.com/cdap-guides/cdap-cube-guide/develop/README.rst

Start with "Share and Discuss!"